package.json

```diff
@@ -71,16 +72,10 @@
     "concurrently": "^6.4.0",
     "cross-env": "^5.2.0",
     "dpdm": "^3.5.0",
-    "eslint": "9.3.0",
-    "eslint-config-react-app": "7.0.1",
-    "eslint-define-config": "2.1.0",
-    "eslint-plugin-flowtype": "8.0.3",
-    "eslint-plugin-import": "2.29.1",
-    "eslint-plugin-jsx-a11y": "6.8.0",
-    "eslint-plugin-lodash": "8.0.0",
-    "eslint-plugin-react": "7.34.1",
-    "eslint-plugin-react-hooks": "4.6.2",
-    "eslint-plugin-storybook": "0.8.0",
+    "eslint": "9.5.0",
+    "eslint-plugin-react": "^7.34.3",
+    "eslint-plugin-react-hooks": "^5.1.0-rc-fb9a90fa48-20240614",
```

Comment:
Do we want a RC used in prod?
Probably not critical since it's a dev dep, but just raising

Reply:
No we don't, but there is a bug in the last stable that just breaks eslint, it fails with an exception.
So we need to update this when possible.